openrpc.json

```diff
@@ -283,6 +283,73 @@
         }
       ]
     },
+    {
+      "name": "wallet_revokePermissions",
+      "tags": [
+        {
+          "$ref": "#/components/tags/MetaMask"
+        }
+      ],
+      "summary": "Revoke existing permissions.",
+      "description": "Revoke permissions from the user",
```

Comment:
you mean from the Ðapp?

Reply:
@danfinlay nice catch, I just updated the description in this [commit](https://github.com/MetaMask/api-specs/pull/145/commits/c9ed7b89796857626ddf9e28d9759b6a936e64f7).

Reply:
@danfinlay yes thats a cool feature, will make sure to adopt this approach for small changes in future, seems more practical. for this thread, it seems this feature is not enabled for the following reason _"Suggestions cannot be applied on outdated comments."_